gapil/bapi/bapi.proto

```diff
@@ -848,6 +855,7 @@ message Statement {
     uint64 slice_assign = 19;   // SliceAssign ID
     uint64 switch = 20;         // Switch ID
     uint64 write = 21;          // Write ID
+    uint64 print = 22;          // Print ID
```

Comment:
Why are you putting the ID at the end (thus breaking the nice alphabetic order, which is fine) but in the previous addition to the Node enum you prefer to insert the new print id in the middle of the list?

Reply:
Done.